webcompat/templates/home-page/form.html

```diff
@@ -62,7 +62,8 @@ <h3 class="wc-Title--s">Report Site Issue</h3>
             </span>
           </div>
           <div class="wc-Form-group js-Form-group">
-            {{ form.steps_reproduce.label(class_='wc-Form-label') }} {{ form.steps_reproduce(class_='wc-Form-item wc-Form-item--textarea js-autoExpand', rows=4) }}
+            <label class="wc-Form-label" for="steps_reproduce">How did you get there?</label>
+            <textarea class="wc-Form-item wc-Form-item--textarea js-autoExpand" id="steps_reproduce" name="steps_reproduce" rows="4" placeholder="Steps to reproduce the issue"></textarea>
```

Comment:
I've tried to include line breaks like:
* `\n`
* `&lt;br&gt;`
* `<br>`

none of these has worked. there is an older SO article:
https://stackoverflow.com/questions/7189223/can-you-have-multiline-html5-placeholder-text-in-a-textarea

but it doesn't seem consistent across browsers. do you still want me to investigate this further?